cms_form/models/cms_form.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Simone Orsi
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models
```

Comment:
A little nit picky but these imports are not ordered alphabetically [as OCA guidelines ask](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports). Perhaps you'd consider importing in a single line?

`from  openerp import _, exceptions, models`